python/cudf/cudf/_lib/stream_compaction.pyx

```diff
@@ -11,10 +11,10 @@ from cudf._lib.column cimport Column
 from cudf._lib.cpp.column.column_view cimport column_view
 from cudf._lib.cpp.stream_compaction cimport (
     apply_boolean_mask as cpp_apply_boolean_mask,
-    distinct_count as cpp_distinct_count,
-    drop_duplicates as cpp_drop_duplicates,
     drop_nulls as cpp_drop_nulls,
     duplicate_keep_option,
+    sort_and_drop_duplicates as cpp_drop_duplicates,
+    unordered_distinct_count as cpp_distinct_count,
```

Comment:
Please keep these Cython import names consistent with the C++ names. Mixing the old and new names makes it more difficult to identify what is being called in libcudf.
```suggestion
    sort_and_drop_duplicates as cpp_sort_and_drop_duplicates,
    unordered_distinct_count as cpp_unordered_distinct_count,
```